environments/dev/manifests/test-ca-tls.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  annotations:
+    service.beta.openshift.io/inject-cabundle: "true"
+  name: test-ca-tls
```

Comment:
I find this a bit odd. Why are we distributing this as part of the dev manifests?

Reply:
You mean, a config map that contains the service CA that signed dex' cert, or a config map with `test` in the name?

Reply:
a configmap with test in the name; this is indicates to me that it's not supposed to be deployed to production, yet out production infrastructure will inherit this.

Reply:
The configmap name is derived from the client_id for the auth, which is currently set to `test`.